### ISSUE TITLE
TypeError: Descriptors cannot not be created directly

### ISSUE THREAD

**Issue by ginkel** [tbowmo/chromecastcontrol#41]
Hi there,

on a fresh install chrome2mqtt exits with the following error:

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/src/app/chrome2mqtt/__main__.py", line 4, in <module>
    from chrome2mqtt.chrome2mqtt import main_loop
  File "/usr/src/app/chrome2mqtt/chrome2mqtt.py", line 18, in <module>
    from chrome2mqtt.devicecoordinator import DeviceCoordinator
  File "/usr/src/app/chrome2mqtt/devicecoordinator.py", line 6, in <module>
    import pychromecast
  File "/usr/local/lib/python3.7/site-packages/pychromecast/__init__.py", line 15, in <module>
    from . import socket_client
  File "/usr/local/lib/python3.7/site-packages/pychromecast/socket_client.py", line 24, in <module>
    from . import cast_channel_pb2
  File "/usr/local/lib/python3.7/site-packages/pychromecast/cast_channel_pb2.py", line 36, in <module>
    type=None),
  File "/usr/local/lib/python3.7/site-packages/google/protobuf/descriptor.py", line 755, in __new__
    _message.Message._CheckCalledFromGeneratedFile()
TypeError: Descriptors cannot not be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).

More information: https://developers.google.com/protocol-buffers/docs/news/2022-05-06#python-updates
```

This must be due to a transitive dependency change that happened within the last ~ 48 hours...

Thanks,
Thilo

**Comment by tbowmo** [comment]
pychromecast got a couple of major bumps in version, so I've bumped the requirements in this project.
Could you try the fixes in #42 ? (It's running on my rpi, so it seems to work..)

**Comment by ginkel** [comment]
LGTM, deployed a new Docker image w/ your fixes from #42 and everything is working again as expected. So I think this can be merged to `master`.

Thanks for the speedy fix!

**Comment by tbowmo** [comment]
merged :) And thanks for reporting the issue
